### DISCUSSION TITLE
Share your recommendations for Font Family

### DISCUSSION THREAD

**Discussion by 3xp11i** [prakhartiwari0/my-web-shortcuts#25] [Ideas]
This is a Discussion about which Font to use for this Extension. Share your recommended ones and **please include a Screenshot where you have implemented that Font everywhere** (Popup, Options, Element selector, Keyboard Shortcut selector).

In their recently merged PR by @beingsie (#14) for the options page, they used [Inter](https://fonts.google.com/specimen/Inter) font for the Options page. You can see how it looks in the comments of the PR.

But I'm not sure if we should use that Font or some other. I am not even sure whether to use one Font universally or multiple Fonts at various places.

 Please share your opinions and ideas :smile:

**Comment by spacesie** [comment]
I can create a mockup of an overall design for this in **[Figma](https://www.figma.com)**.

I would recommend starting of with **[Inter](https://fonts.google.com/specimen/Inter)** to create a quick mockup and starting base design and move on to a different font when we alter the design in general.

Please checkout this link, it showcases different designs that use Inter and how other fonts can be paired with it!
- [Inter Font Pairing & Alternatives](https://maxibestof.one/typefaces/inter)

**Comment by spacesie** [comment]
# Mockup of UI mockup

## Fonts used
- Inter
- Font Awesome

## Color scheme
- I went with a basic color scheme for now to give space for the decision of the layout.

## Changes suggested included in the UI redesign
- For better UX the following changes were implemented in the design

  - Both `disable` buttons were changed to **toggles**.
  - The `on` state in the **added shortcuts** list was changed to a **toggle**.
  - Several elements have had their text changed for a more clear and concise UX.
  
  **Important change:**
  - The `on` state when **selecting** a shortcut was removed. I believe having this option while selecting a key for the shortcut is unnecessary/low importance, as it is available in the shortcut list where it makes more sense having it on.
    - **Note**: As a first time user when I saw this option I was confused on what this did. Once seeing it on the shortcut list I had made the connection.
    
## Screenshot of UI mockup

<img width="1453" alt="image" src="https://github.com/prakhartiwari0/my-web-shortcuts/assets/140430987/b699e5e1-aa53-4f94-9e5f-7456bc434dab">

**Reply by 3xp11i** [reply]
This looks awesome!

**Reply by 3xp11i** [reply]
Can you share the mockup Figma link? I guess it's designed on Figma only right?

**Comment by 3xp11i** [comment]
>   **Important change:**
>   
>   * The `on` state when **selecting** a shortcut was removed. I believe having this option while selecting a key for the shortcut is unnecessary/low importance, as it is available in the shortcut list where it makes more sense having it on.
>     
>     * **Note**: As a first time user when I saw this option I was confused on what this did. Once seeing it on the shortcut list I had made the connection.

I guess that button is causing confusion. It is actually not to enable/disable the shortcut (which is done by the shortcuts list in the popup), but to enable/disable element selection on the website. Try it out yourself or you can watch the [Demo video](https://www.youtube.com/watch?v=x_fYfporwmA).

**Reply by 3xp11i** [reply]
I should mention that pressing the `CTRL` does the same thing. So we can put that switch with a little text in the bottom or somewhere near it which says, "Keep the `CTRL` key pressed to disable Element Selection"

**Reply by spacesie** [reply]
Thanks for clarifying!

**Comment by spacesie** [comment]
> Can you share the mockup Figma link? I guess it's designed on Figma only right?

Yes, the initial design was created using Figma. I am working on cleaning up the file and will send the link when ready!
- One issue would be visiting from link will be READ only.
- Would you prefer the file itself?

**Reply by 3xp11i** [reply]
Read only will work, only the element selector needs a little change that I mentioned above, rest is okay for now. There will be many more features and options coming in the future so the UI will change accordingly.

**Reply by spacesie** [reply]
Sounds great!
- [Figma Link](https://www.figma.com/file/BgoB8sOUVOOW1pxpJol5vW/My-Web-Shortcuts?type=design&node-id=0-1&mode=design)

**Comment by 3xp11i** [comment]
@madebysergio hey buddy, I see that you have deleted all your social profiles, and I can't find a way to connect with you, if you ever see this message, please do reach out to me.

**Reply by spacesie** [reply]
Hi @prakhartiwari0! Long time! How are you? Let me know what's up!

**Reply by 3xp11i** [reply]
Damn never expected you would reply, I thought you were ~~dead~~ gone from the internet 😂 😓 

I am going to launch the premium version of My Web Shortcuts. would you like to give it a try? Message me on discord, can't share the username publicly, so join this server if you don't know my username: [browsergadgets.io/discord](https://browsergadgets.io/discord)

**Reply by 3xp11i** [reply]
Yes, and also check out the ui of https://browsergadgets.io/, it is still an early project and a lot of things need to be added and worked on.
